### ISSUE TITLE
Guild view not able to handle np.inf

### ISSUE THREAD

**Issue by mherlich** [guildai/guildai#241]
`guild view` is unable to handle `np.inf`in tensorflow scalars. It does not show any runs when a single run contains `np.inf`. `guild runs` shows the runs as expected. Here is an example to reproduce the problem:
```
import numpy as np
import tensorflow as tf

writer = tf.summary.create_file_writer("log")
with writer.as_default():
    tf.summary.scalar("Name", np.inf ,step=0)
    writer.flush()
```

Additional information: guild_version:             0.7.0.post1

**Comment by gar1t** [comment]
Thanks for the report! I create a resolution doc for this to help confirm the fix when it lands:

https://github.com/guildai/issue-resolution/tree/master/241-guild-view-not-able-to-handle-np-inf

**Comment by gar1t** [comment]
It looks like this was fixed back in 0.7.1. For steps to verify, see the [issue doc](https://github.com/guildai/issue-resolution/tree/master/241-guild-view-not-able-to-handle-np-inf).
